content/2_Bronze/Complete_Rec.mdx

```diff
@@ -77,7 +77,7 @@ ll solve(int i, ll s1, ll s2){
 		return abs(s1 - s2);
```

Comment:
i also modified this file in pr 1652, hopefully the conflicts won't be too bad...